packages/main/src/ColorPalettePopover.ts

```diff
@@ -178,12 +178,12 @@ class ColorPalettePopover extends UI5Element {
 
 	onAfterOpen() {
 		const colorPalette = this._colorPalette;
-		if (colorPalette.showDefaultColor && !colorPalette._currentlySelected) {
-			colorPalette.colorPaletteNavigationElements[0].focus();
-		} else if (colorPalette._shouldFocusRecentColors && colorPalette.showRecentColors) {
-			colorPalette.recentColorsElements[0].focus();
-		} else {
+		if (colorPalette._currentlySelected) {
```

Comment:
The navigation works good now. The only thing that's left, is that when we select a color from the `More Colors` color-picker, the selected color (as per UI5 behaviour) should be selected and focused in the `recentColors` section, after the color is being selected and the color-picker is closed. 
Right now it only gets focused.

Reply:
Done